frontend/src/interface/processBackendData.js

```diff
@@ -67,10 +67,10 @@ export function processBackendData(data, callback) {
         }
     }
 
-    setMaterialCompositionLabels(materialMap.keys());
-    setMaterialCompositionData(materialMap.values());
-    setImpactAssessmentData(impactMap);
-    setColumnChartData(assessmentDataInPercent);
+    setMaterialCompositionLabels(scenarioName, materialMap.keys());
+    setMaterialCompositionData(scenarioName, materialMap.values());
+    setImpactAssessmentData(scenarioName, impactMap);
+    setColumnChartData(scenarioName, assessmentDataInPercent);
 
     callback();
```

Comment:
@ScoutAtlas and this is probably about the point where we should store the processed calculation data.